src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -99,7 +99,7 @@ function getBox(volume, inverseTileTransform) {
 
 const rePosition = new RegExp('gl_Position.*(?![^]*gl_Position)');
 const reMain = new RegExp('[^\\w]*main[^\\w]*(void)?[^\\w]*{');
-function patchMaterial(material) {
+export function patchMaterial(material) {
```

Comment:
Maybe you could have moved this function to its own utility file.
At the very least change its name to really reflect what it's doing (`patchMaterialForLogDepthSupport`?)